packages/adblocker-webextension/test/index.test.ts

```diff
@@ -125,3 +131,213 @@ describe('#fromWebRequestDetails', () => {
     });
   });
 });
+
+describe('html-filtering', () => {
+  context('#shouldApplyReplaceSelectors', () => {
+    function createWebRequestDetails({
+      responseType,
+      headers,
+    }: {
+      responseType: WebRequest.ResourceType;
+      headers: Record<string, string>;
+    }) {
+      const url = 'https://foo.com/';
+      const requestId = 'req-01';
+      const tabId = 1;
+      const details: OnHeadersReceivedDetailsType = {
+        requestId,
+        tabId,
+        url,
+        responseHeaders: Object.entries(headers).map(([name, value]) => ({
+          name,
+          value,
+        })),
+        type: responseType,
+      };
+
+      return details;
+    }
+
+    const sets: Array<Parameters<typeof createWebRequestDetails>[0] & { result: boolean }> = [
+      {
+        responseType: 'main_frame',
+        headers: {
+          'content-type': 'text/html',
+        },
+        result: true,
+      },
+      {
+        responseType: 'script',
+        headers: {
+          'content-type': 'application/javascript',
+        },
+        result: true,
+      },
+      {
+        responseType: 'stylesheet',
+        headers: {},
+        result: true,
+      },
+      {
+        responseType: 'script',
+        headers: {
+          'content-type': 'application/javascript',
+          'content-length': (MAXIMUM_RESPONSE_BUFFER_SIZE + 1).toString(),
+        },
+        result: false,
+      },
+    ];
+
+    sets.forEach((set) => {
+      const details = createWebRequestDetails(set);
+      const request = fromWebRequestDetails(details);
```

Comment:
please move this into `it` - otherwise those are initiated when tests are being prepared, not run, which makes tests selected by `only` to run this code.

Reply:
Updated in [`9991a4a` (#14)](https://github.com/seia-soto/adblocker/pull/14/commits/9991a4a7979f149105991bd4b199be1250a86d41)